api/src/Validator/AssertJsonSchemaValidator.php

```diff
@@ -15,6 +15,10 @@ public function validate($value, Constraint $constraint) {
             throw new UnexpectedTypeException($constraint, AssertJsonSchema::class);
         }
 
+        if (null === $value) {
```

Comment:
Wieso ist ```$value === null``` automatisch ok?

Reply:
Ist ein Konzept was Symfony bei den eigenen Validators auch so einsetzt: `null`-Wert wird normalerweise als ok angesehen (bzw. vom Validator nicht überprüft) und man steuert separat via `Assert\IsNull` oder `Assert\NotNull`, wenn man null-Wert explizit vermeiden oder erzwingen will.

So is es nun möglich, dass wir null-Werte beim Erstellen erlauben, und in diesem Fall im DataPersister mit einem Default-Wert befüllen.